src/App.jsx

```diff
@@ -2,11 +2,74 @@ import { useState } from "react";
 import "./App.css";
 
 function City(props) {
-  return <div>This is the City component</div>;
+  return (
+    <>
+      <div>
+        <h2>
+          {props.City}, {props.State}
+        </h2>
+        <ul>
+          <li>State: {props.State}</li>
+          <li>
+            Location: ({props.Lat}, {props.Long})
+          </li>
+          <li>Population (estimated): {props.EstimatedPopulation}</li>
+          <li>Total Wages: {props.TotalWages}</li>
+        </ul>
+      </div>
+    </>
+  );
 }
 
 function ZipSearchField(props) {
```

Comment:
This function should really just be the searchbox. It shouldn't really contain the cities.